implementations/exec.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+from code_df import (
+    code_changes_git,
+    code_changes_lines,
+    reviews,
+    reviews_accepted,
+    )
+
+from scripts import (
+    code_changes_git,
+    code_changes_lines,
+    reviews,
+    reviews_accepted)
+
+import utils
+
+
+Git_metrics = [code_changes_git, code_changes_lines]
+GitHub_metrics = [reviews, ]
+
+
+def write_results(file=None, Git_results=None, GitHub_results=None):
+    print(Git_results)
+    print(GitHub_results)
+
+
+def test_git_metrics(items_file, date_range, is_code=None, conds=None):
+
+    items = utils.read_json_file(items_file)
+    Git_results = {}
+
+    for metric in Git_metrics:
+        metric_obj = metric(items, date_range)
+        Git_results[str(metric)] = metric_obj.compute()
+
+    return Git_results
+
+
+def test_github_metrics(items_file, date_range):
+    items = utils.read_json_file(items_file)
+    GitHub_results = {}
+
+    for metric in GitHub_metrics:
+        metric_obj = metric(items, date_range)
+        GitHub_results[str(metric)] = metric_obj.compute()
+
+    return GitHub_results
+
+
+def main(use_pandas=False, include_timeseries=False, git=True, github=False, ):
+
+    date_since = datetime.strptime("2018-09-07", "%Y-%m-%d")
+    Git_results = test_git_metrics('git-commits.json', (date_since, None))
+    write_results(Git_results=Git_results)
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
It would be great to add before `main()` the possibility to parse arguments from command line. In case you need an example to use `argparse, you can check: https://gist.github.com/valeriocos/f75e8081c9ca121851a2fb7def93d43b#file-scava_stats-py-L52

Reply:
Thanks valeriocos. Yes, the final version will use command line args.